src/settings.ini

```diff
@@ -1,9 +1,9 @@
 [COMMON]
-save_model_dir = "project/model_dir/"
-log_dir = "project/log_dir"
+save_model_dir = "/rfanfs/pnl-zorro/home/sq566/pycharm/Suheyla/data/comp/training"
```

Comment:
@SenthilCaesar , please bring back the discrete config file I pushed. Again, we need to be careful about not committing every local changes.